docs/1 Introduction/1 Introduction.md

```diff
@@ -1,45 +1,56 @@
 # An introduction to Starport
 
-Let us dive into Starport, what we can achieve with it, and which other technologies play well with it.
+Starport is a developer-friendly interface to the Cosmos SDK.
 
 ## What is Starport?
 
-Starport is a tool that makes it easier to create blockchains.
+The Starport tool is the easiest way to create blockchains using the Tendermint Consensus engine and the Cosmos SDK. <!-- @fadeev does it matter that we use Go? --> Starport is written in the Go programming language. The new blockchain has a Proof-of-Stake (PoS) system with validators that can be defined in the genesis block.
 
-Starport uses the Tendermint Consensus engine and the Cosmos SDK to create a blockchain application in the Go programming language. This blockchain has a Proof-of-Stake system with validators that can be defined in the genesis block.
+With just a few commands, you can use Starport to:
 
-With just a few commands, you can create a blockchain, launch it, serve it on the cloud and have a GUI ready to start testing your application.
+1. Create a blockchain
+2. Launch a blockchain
+3. Serve your blockchain on the cloud
+4. Create a client-side app you can interact with in web browser
 
-Bootstrapping blockchains was initially the job of the `scaffold` program, which was used to create a blockchain application. Starport takes it to the next level and also creates a user interface with Vue.js, which provides a good starting point for developers creating a browser-based client-side application for your blockchain.
+<!-- do we need to mention Vue.js? -->
 
-The scaffolded application still includes a command line interface that lets you manage keys, create validators, send tokens.
+ Starport creates a user interface with Vue.js. This user interface provides a good starting point for developers by creating a browser-based client-side app for your blockchain. The scaffolded app created with Starport includes a command line interface that lets you manage keys, create validators, send tokens.
 
-## Projects using Tendermint / Cosmos SDK
+Note: Starport replaces the `scaffold` program that was previously used to create a blockchain app.
 
-There are many projects already showcasing that the Tendermint BFT Consensus Engine and the Cosmos SDK.
+## Projects using Tendermint and Cosmos SDK
 
-The following projects are using the technology:
+Many projects already showcase the Tendermint BFT Consensus Engine and the Cosmos SDK, including:
```

Comment:
We actually have projects using Starport itself, which is pretty nice. Some of them are real chain launched into production, other are interesting projects.

https://github.com/Sifchain/sifnode
https://github.com/crypto-org-chain/chain-main (initialized with Starport: https://github.com/crypto-org-chain/chain-main/commit/37b2ecb49a9aae7c581270a4f2dbecfcd8e8a6e9)
Used in the demo of Interchain accounts: https://github.com/chainapsis/cosmos-sdk-interchain-account
https://github.com/EG-easy/affondra
https://github.com/chantmk/Finding-imposter

Reply:
We also have this beautiful website now, that replaces the need for listing too many projects https://cosmos.network/ecosystem

Reply:
And eventually we'll have `cosmos.network/starport`, so that there is a dedicated bragging section for the CLI 😄